crates/bevy_core_pipeline/src/auto_exposure/auto_exposure.wgsl

```diff
@@ -0,0 +1,172 @@
+#import bevy_render::view::View
```

Comment:
It would be nice to have a brief summary of the algorithm / goals at the top of this shader: all of the heavy lifting seems to be done here, so it seems like a natural home for that sort of material.